MICP_Assignments/Assignment_Week_4.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MICP_Assignments
+{
+    public class Assignment_Week_4
+    {
+        /*

```

Comment:
Thanks for commenting! It's very helpful for your reviewers + great practice for when you'll have to talk through your logic with your interviewer.